charts/kong/example-values/quickstart-enterprise-licensed-aio.yaml

```diff
@@ -0,0 +1,291 @@
+      valueFrom:
+        secretKeyRef:
+          key: password
+          name: kong-config-secret
+    kong_admin_url: https://localhost:8444
+    kong_workspace: default
+    publish_service: kong/quickstart-kong-proxy
+  image:
+    repository: docker.io/kong/kubernetes-ingress-controller
+    tag: "2.2"
+  ingressClass: default
+  installCRDs: false
+manager:
+  annotations:
+    konghq.com/protocol: https
+  enabled: true
+  http:
+    containerPort: 8002
+    enabled: false
+    servicePort: 8002
+  ingress:
+    annotations:
+      konghq.com/https-redirect-status-code: "301"
+      kubernetes.io/ingress.class: default
+      nginx.ingress.kubernetes.io/backend-protocol: HTTPS
+    enabled: true
+    hostname: kong.127-0-0-1.nip.io
+    path: /
+    tls: quickstart-kong-admin-cert
+  tls:
+    containerPort: 8445
+    enabled: true
+    parameters:
+    - http2
+    servicePort: 8445
+  type: ClusterIP
+migrations:
+  enabled: true
+  postUpgrade: true
+  preUpgrade: true
+namespace: kong
+podAnnotations:
+  kuma.io/gateway: enabled
+portal:
+  annotations:
+    konghq.com/protocol: https
+  enabled: true
+  http:
+    containerPort: 8003
+    enabled: false
+    servicePort: 8003
+  ingress:
+    annotations:
+      konghq.com/https-redirect-status-code: "301"
+      konghq.com/protocols: https
+      konghq.com/strip-path: "false"
+      kubernetes.io/ingress.class: default
+    enabled: true
+    hostname: developer.127-0-0-1.nip.io
+    path: /
+    tls: quickstart-kong-portal-cert
+  tls:
+    containerPort: 8446
+    enabled: true
+    parameters:
+    - http2
+    servicePort: 8446
+  type: ClusterIP
+portalapi:
+  annotations:
+    konghq.com/protocol: https
+  enabled: true
+  http:
+    enabled: false
+  ingress:
+    annotations:
+      konghq.com/https-redirect-status-code: "301"
+      konghq.com/protocols: https
+      konghq.com/strip-path: "true"
+      kubernetes.io/ingress.class: default
+      nginx.ingress.kubernetes.io/app-root: /
+    enabled: true
+    hostname: developer.127-0-0-1.nip.io
+    path: /api
+    tls: quickstart-kong-portal-cert
+  tls:
+    containerPort: 8447
+    enabled: true
+    parameters:
+    - http2
+    servicePort: 8447
+  type: ClusterIP
+postgresql:
+  enabled: true
+  global:
+    postgresql:
+      database: kong
+      existingSecret: kong-config-secret
+      username: kong
+proxy:
+  annotations:
+    prometheus.io/port: "9542"
+    prometheus.io/scrape: "true"
+  enabled: true
+  http:
+    containerPort: 8080
+    enabled: true
+    hostPort: 80
+  ingress:
+    enabled: false
+  labels:
+    enable-metrics: true
+  tls:
+    containerPort: 8443
+    enabled: true
+    hostPort: 443
+  type: LoadBalancer
+replicaCount: 1
+secretVolumes: []
+status:
+  enabled: true
+  http:
+    containerPort: 8100
+    enabled: true
+  tls:
+    containerPort: 8543
+    enabled: false
+updateStrategy:
+  rollingUpdate:
+    maxSurge: 100%
+    maxUnavailable: 100%
+  type: RollingUpdate
```

Comment:
I would suggest since we're here we add some level of testing for this example if it's not too much trouble. Ideally we would run this new example through our integration test suite, and verify the deployment at least at a basic level to help avoid future regressions.